harvard-lts/epadd-curator-app
# Reject Batches with Invalid Characters

**Rejects the batch and sends an email**
* * *

**Ticket Link**: https://jira.huit.harvard.edu/browse/LTSEPADD-117

# How should this be tested?
1. Add the two new variables to your already existing .env (or set up a new .env from env-example.txt):
```
RELATIVE_DRS_CONFIG_PATH=data/blobs/sidecarfiles/drsConfig.txt
INVALID_CHARACTERS_MESSAGE="Invalid characters were found in the export.  Please see documentation (forthcoming) on what to do when this occurs."
```

2. Run the tests as described here: https://github.com/harvard-lts/epadd-curator-app#running-tests

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? yes
- integration tests? no
